scripts/run-image

```diff
@@ -133,7 +129,8 @@ if __name__ == '__main__':
             'volumes': '',
             'ports': '',
             'cmd': args.command or '',
-            'init': init
+            'init': '--init',
```

Comment:
Did we decide that we always want `--init`?

Reply:
Conversation offline with Mike on this one...